File: BOJ/Kotlin/그리디 알고리즘/4796.kt
import java.io.BufferedReader
import java.io.InputStreamReader
import kotlin.math.min

fun main() = with(BufferedReader(InputStreamReader(System.`in`))) {
    var count = 1

    while (true){
        val (L, P, V) = readLine().split(' ').map { it.toInt() }
        var res = 0

        if (L == 0 && P == 0 && V ==0) break

        res = (V/P) * L + min((V%P), L)

        println("Case ${count}: ${res}")

        count++
    }
}

File: BOJ/Python/그리디 알고리즘/4796.py
count = 1
while True:
    L, P, V = map(int, input().split())
    if L == 0 and P == 0 and V == 0:
        break

    res = 0

    while V > 0:
        if V > L:
            res += L
        else:
            res += V
        V -= P

    print("Case {0}: {1}".format(count, res))

    count += 1
